websocket plugin needs a bunch of checkouts more

obs browser requires building cef (chromium) as a submodule

aja exists but isn't friendly to package (no soversion), and is just a plugin
for some fancy enterprise hardware

diff --git a/plugins/CMakeLists.txt b/plugins/CMakeLists.txt
index f928f77..0cde953 100644
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -153,7 +153,6 @@ elseif(OS_LINUX)
   add_subdirectory(sndio)
   add_subdirectory(obs-vst)
 
-  check_obs_browser()
 elseif(OS_FREEBSD)
   add_subdirectory(linux-capture)
   add_subdirectory(linux-pipewire)
@@ -178,7 +177,6 @@ endif()
 if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/obs-websocket/CMakeLists.txt)
   add_subdirectory(obs-websocket)
 else()
-  obs_status(FATAL_ERROR "obs-websocket submodule not available.")
 endif()
 
 add_subdirectory(image-source)
@@ -190,4 +188,3 @@ add_subdirectory(obs-filters)
 add_subdirectory(obs-transitions)
 add_subdirectory(rtmp-services)
 add_subdirectory(text-freetype2)
-add_subdirectory(aja)
